doc/reference/dune/alias.rst

```diff
@@ -1,55 +1,58 @@
+.. highlight:: dune
+
 alias
 -----
 
-The ``alias`` stanza adds dependencies to an alias or specifies an action to run
-to construct the alias.
+.. describe:: (alias ...)
+
+   Add dependencies to an alias.
+
+   The typical use of the ``alias`` stanza is make an alias depend on other ones::
+
+       (alias
+        (name runtest)
+        (deps
+         (alias test-unit)
+         (alias test-integration)))
+
+   In previous versions of the dune language, it was also possible to specify
+   an action to run to construct the alias. Please use a :doc:`rule` stanza
+   with the ``alias`` field instead.
+
+   .. describe:: (name <name>)
 
-The syntax is as follows:
+      An alias name such as ``runtest``.
 
-.. code:: dune
+      This field is required.
```

Comment:
Maybe `runtest` is not a good example because this can create confusion between `dune runtest` (not an alias but people might think it is) and `dune build @runtest` (an alias). Maybe `install` (`dune install` exists too but isn't widely used) or `check`?

Reply:
I'm not sure. Adding to `runtest` is arguably the most common use of `(alias)`, for example to depend on `.exe` file to be built but not executed. In contrast, adding to `install` does nothing (since it won't generate install entries in the `.install` file).

Reply:
Then maybe explaining that this will modify what runs as part of `dune build @runtest` (which is equivalent to `dune runtest`) will clear things up.

The thing I want to avoid is for users to think that aliases turn into subcommands.